CMakeLists.txt

```diff
@@ -224,27 +215,28 @@ add_custom_target(genivi-tarball
   COMMAND tar -cz -C /tmp/GENIVI -f ${CMAKE_BINARY_DIR}/genivi.tar.gz .
 )
 
-project (${PROJECT})
-
-#ADD_DEPENDENCIES(${PROJECT} Policy)
-
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
 # Please do not change compiler/linker flags if You do not know how particular
 # flag is handled by CMake
 set(CMAKE_INSTALL_PREFIX ${CMAKE_CURRENT_BINARY_DIR})
 set(ARCHIVE_OUTPUT_DIRECTORY ./bin)
 
-set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++0x -Wall -Werror -Wuninitialized -Wvla")
+# Set default cxx_flags (if not defined in toolchain files)
+if (NOT CMAKE_CXX_FLAGS)
+  set(CMAKE_CXX_FLAGS "-fPIC -std=gnu++0x -Wall -Werror -Wuninitialized -Wvla")
+endif()
+
+set(CMAKE_CXX_FLAGS_RELEASE " -DNDEBUG -s -O2")
+set(CMAKE_CXX_FLAGS_DEBUG   " -DDEBUG -g3 -ggdb3")
 
 if (USE_GOLD_LD)
-  execute_process(COMMAND ld -v OUTPUT_VARIABLE result)
-  string(FIND ${result} Binutils GOLD_LD_FOUND)
-  if (GOLD_LD_FOUND EQUAL -1)
-    message(SEND_ERROR "Gold linker not found!")
-  else()
+  execute_process(COMMAND ${CMAKE_CXX_COMPILER} -fuse-ld=gold -Wl,--version ERROR_QUIET OUTPUT_VARIABLE GOLD_VERSION)
+  if(GOLD_VERSION MATCHES "GNU gold")
     message(INFO " Use gold linker")
```

Comment:
@mkorniichuk replace INFO to status. There is no such message threshold as INFO

Reply:
@LuxoftAKutsan fixed within a89e4042f9a29e6b3ec306046c57bd2bd2e2c4f7